Tests/SentryTests/Integrations/SentryPerformanceTrackerTests.swift

```diff
@@ -0,0 +1,219 @@
+import XCTest
+
+class SentryPerformanceTrackerTests: XCTestCase {
+    
+    private static let dsnAsString = TestConstants.dsnAsString(username: "SentryPerformanceTrackerTests")
+    
+    private class Fixture {
+
+        let someTransaction = "Some Transaction"
+        let someOperation = "Some Operation"
+        
+        func getSut(initSDK: Bool = false) -> SentryPerformanceTracker {
+            if initSDK {
+                SentrySDK.start { options in
+                    options.dsn = SentryPerformanceTrackerTests.dsnAsString
+                    options.debug = true
+                    options.diagnosticLevel = SentryLevel.debug
+                }
+            }
+            return  SentryPerformanceTracker()
+        }
+
+        func getSpans(tracker: SentryPerformanceTracker) -> [SpanId: Span] {
+            let result = Dynamic(tracker).spans as [SpanId: Span]?
+            return result!
+        }
+        
+        func getStack(tracker: SentryPerformanceTracker) -> [Span] {
+            let result = Dynamic(tracker).activeStack as [Span]?
+            return result!
+        }
+    }
+    
+    private var fixture: Fixture!
+    
+    override func setUp() {
+        
+        fixture = Fixture()
+    }
+    
+    override func tearDown() {
+        SentrySDK.close()
+    }
+    
+    func testSingleton() {
+        XCTAssertEqual(SentryPerformanceTracker.shared(), SentryPerformanceTracker.shared())
+    }
+    
+    func testStartSpan_CheckScopeSpan() {
+        let tracker = fixture.getSut(initSDK: true)
```

Comment:
`l`: Please rename this everywhere in this class. Then you always know what the subject under test (sut) is.
```suggestion
        let sut = fixture.getSut(initSDK: true)
```